base/src/commonMain/kotlin/utils/Decimal.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.base.utils
+
+expect class Decimal
+
+expect operator fun Decimal.plus(value: Decimal): Decimal
+expect fun Decimal.plus(value: Decimal, scale: Int): Decimal
+expect fun Decimal.plus(
+    value: Decimal,
+    scale: Int,
+    roundingMode: RoundingMode = RoundingMode.RoundHalfEven
+): Decimal
+
+expect operator fun Decimal.minus(value: Decimal): Decimal
+expect fun Decimal.minus(value: Decimal, scale: Int): Decimal
+expect fun Decimal.minus(
+    value: Decimal,
+    scale: Int,
+    roundingMode: RoundingMode = RoundingMode.RoundHalfEven
+): Decimal
+
+expect operator fun Decimal.div(value: Decimal): Decimal
+expect fun Decimal.div(value: Decimal, scale: Int): Decimal
+expect fun Decimal.div(
+    value: Decimal,
+    scale: Int,
+    roundingMode: RoundingMode = RoundingMode.RoundHalfEven
+): Decimal
+
+expect operator fun Decimal.times(value: Decimal): Decimal
+expect fun Decimal.times(value: Decimal, scale: Int): Decimal
+expect fun Decimal.times(
+    value: Decimal,
+    scale: Int,
+    roundingMode: RoundingMode = RoundingMode.RoundHalfEven
+): Decimal
+
+sealed class RoundingMode {
+    object RoundDown : RoundingMode()
+    object RoundHalfEven : RoundingMode()
+    object RoundUp : RoundingMode()
+}
+
+expect fun toPlatformSpecificRoundCode(roundingMode: RoundingMode): Int
```

Comment:
This doesnt need to be in the common level. Its a platform detail so it can just live in iosMain and androidMain without being exposed to common code through expect/actuals